.github/workflows/tests.yml

```diff
@@ -21,6 +21,18 @@ jobs:
     steps:
       - name: Check out repository
         uses: actions/checkout@v3
+
+      - name: Setup cache
+        id: cache-test-data
+        uses: actions/cache@v3
+        with:
+          path: test_data
+          key: test_data
+
+      - name: Copy test data
+        if: steps.cache-test-data.outputs.cache-hit == 'true'
+        run: cp -r test_data/ ${GITHUB_WORKSPACE}/tdms/tests/system/data
```

Comment:
Why not simply caching `${GITHUB_WORKSPACE}/tdms/tests/system/data`?

Reply:
An excellent question... (blindly copying the example might have had something to do with it)